MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -37,6 +37,7 @@
 
   <ItemGroup>
       <Compile Include="*Test.cs" />
+      <None Remove="ReadOnlyDecoratedList.cs" />
```

Comment:
Is this needed?

Reply:
That's strange; it was supposed to be in the "if .NET Framework" section. I don't even get why it compiles. I'll take a look at that.